src/components/Header/Header.test.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { cleanup, fireEvent, render } from '@testing-library/react';
+import { Router } from 'react-router-dom';
+import { createMemoryHistory } from 'history';
+import { Header } from './Header';
+
+afterEach(cleanup);
+
+const setup = () => {
```

Comment:
you could save setup utility function in separate file (for example utils/tests.ts)